controllers/ProfileController.js

```diff
@@ -0,0 +1,94 @@
+import models from '../database/models';
+import ServerResponse from '../modules';
+
+const { successResponse, errorResponse } = ServerResponse;
+/**
+ * @class ProfileController
+ * @exports ProfileController
+ */
+export default class ProfileController {
+  /**
+   *
+   * @method createProfile
+   * @description Create a profile for registered user
+   * @param {object} req express request object
+   * @param {object} res express response object
+   * @returns {object} returns profile info
+   * @memberof ProfileController
+   */
+  static async createProfile(req, res) {
+    const { avatar, bio } = req.body;
+    const { id } = req.user;
+    const profileExist = await models.Profile.findOne({ where: { userId: id } });
+    if (profileExist) {
+      return errorResponse(res, 409, 'Profile already exist');
+    }
+    await models.Profile.create({
+      userId: id,
+      avatar,
+      bio,
+    });
+    const { firstName, lastName } = req.user;
+    const resData = {
+      message: 'Your profile has been created successfully',
+      profile: {
+        name: `${firstName} ${lastName}`,
```

Comment:
Why is `name` in your response from the request object and not from the database?

Reply:
The req.user is from the user token. I decided to use it instead of getting the user id from req.user and then querying the database to get the names